cli/flox/src/commands/services/stop.rs

```diff
@@ -32,12 +32,19 @@ impl Stop {
 
         for process in named_processes {
             if !process.is_running {
-                message::warning(format!("service '{}' is not running", process.name));
+                message::warning(format!("Service '{}' is not running", process.name));
                 continue;
             }
 
-            stop_services(&socket, &[&process.name])?;
-            message::updated(format!("service '{}' stopped", process.name));
+            if let Err(err) = stop_services(&socket, &[&process.name]) {
+                message::error(format!(
+                    "Failed to stop service '{}': {}",
+                    process.name, err
+                ));
+                continue;
+            }
```

Comment:
Its likely to get the same error message here N times for N services, e.g. if there is an issue communicating with process-compose.
Also returning with success even if we have failed to stop any service appears a bit counterintuitive.